astiav_test.go

```diff
@@ -85,6 +85,43 @@ func (h *helper) inputFormatContext(name string) (fc *FormatContext, err error)
 	return
 }
 
+func (h *helper) bitStreamFilterContext(fc *FormatContext, si int, fn string) (*BitStreamFilterContext, error) {
```

Comment:
I don't feel the way we test `SendPacket` and `ReceivePacket` is sufficient right now but I'm not sure how to do it differently. In the meantime I'd rather not add it in the PR and add a TODO instead.
Therefore could you remove the changes in `astiav_test.go` and in `bit_stream_filter_context_test.go` remove all things needed to test both `SendPacket` and `ReceivePacket`?